mlflow/langchain/langchain_tracer.py

```diff
@@ -61,6 +73,21 @@ def __init__(
         self._parent_span = parent_span
         self._run_span_mapping: Dict[str, LiveSpan] = {}
         self._prediction_context = prediction_context
+        self._request_id = None
+
+    def _dump_trace(self) -> str:
+        """
+        This method is only used to get the trace data from the buffer in databricks
+        serving, it should return the trace in dictionary format and then dump to string.
+        """
+
+        def _default_converter(o):
+            if isinstance(o, datetime):
+                return o.isoformat()
```

Comment:
why only datetime?

Reply:
Because inference_table recorded trace should all contain json-serializable fields except datetime

Reply:
got it!